hangman.c

```diff
@@ -0,0 +1,312 @@
+//Olivier Gabison - Hangman Game
+//NOTE: I don't understand the error here as well?
+
+#include <stdio.h>
+
+/*
+WORD TO GUESS: CHOCOLATE
+NUMBER OF MAXIMUM ERRORS: 5
+*/
+
+int main(){
+
+  int errors = 5;
+  char answer = {'c','h','o','c','o','l','a','t','e','\0'};
```

Comment:
Two of your warnings come from your array initializations. Basically, the compiler thinks your initializing something other than an array, which is why it's giving out a warning. So how do you fix? Think it through... what can you change about your data type initialization so that the compiler is receiving the correct & full information. Let me know what you think!